src/Justimmo/Model/Wrapper/V1/EmployeeWrapper.php

```diff
@@ -124,7 +124,7 @@ protected function mapAttachmentGroup(\SimpleXMLElement $xml, $attachmentAware,
             if (array_key_exists('pfad', $data)) {
                 $data['big'] = $data['pfad'];
 
-                $attachment = new Attachment($data['orig'] ?? $data['pfad'], $type, $group);
+                $attachment = new Attachment($data['pfad'], $type, $group);
```

Comment:
```suggestion
                $attachment = new Attachment(isset($data['orig']) ? $data['orig'] : $data['pfad'], $type, $group);
```
The original logic should be kept, but please fix it like this so it's compatible with php 5.6

Reply:
> This PR would introduce breaking changes because Attachment::getData() won't return an array anymore where the key orig is always present.

How do you want to present orig if it is not present in the API's XML answer?
Attachment::getData() returns an array and contains the very same configs that the API passed.

> The original logic should be kept, but please fix it like this so it's compatible with php 5.6

I don't really understand your explanation. Your solution wouldn't fix the issue described in the ticket.
1. pfad is not equal with orig!
2. for PHP-SDK the pfad is always small and orig is not presented if setPictureSize() method is not used. In the case, because orig is not presented it would get the value from pfad and its value is small. So it would be the same result like it is described in the ticket:
$attachment = new Attachment(isset($data['orig']) ? $data['orig'] : $data['pfad'], $type, $group);